packages/mocks/src/apollo/rateDataMock.ts

```diff
@@ -38,6 +38,11 @@ const rateRevisionDataMock = (data?: Partial<RateRevision>): RateRevision => {
         formData: {
             rateType: 'AMENDMENT',
             rateCapitationType: 'RATE_CELL',
+            rateMedicaidPopulations: [
```

Comment:
This change would apply to a ton of tests (including the RateDetails test that had the failure) since this inside of a reusable test mock. I don't think we want to set `rateMedicaidPopulations` here because this field is would only be populated when the `dsnp` flag is on. The [test branch](https://github.com/Enterprise-CMCS/managed-care-review/compare/test-rate-medicaid-populations?expand=1#diff-1af1a8a6b4ce16a4da87c355122f0f49f518ed20015036035e049daa8971851aR333) I created shows an example of setting `rateMedicaidPopulations` directly in the `rateDetailsSummarySection` test

I still don't see exactly why this caused the test failure we were seeing earlier. I'm curious what @allly-b thinks as far as how much more engineering time to spend on this? My sense is this is